src/qibo/backends/clifford.py

```diff
@@ -150,6 +172,28 @@ def execute_circuit(self, circuit, initial_state=None, nshots: int = 1000):
         Returns:
             :class:`qibo.quantum_info.clifford.Clifford`: Object storing to the final results.
         """
+        from qibo.quantum_info.clifford import Clifford  # pylint: disable=C0415
+
+        if self.platform == "stim":
+            circuit_stim = self._stim.Circuit()  # pylint: disable=E1101
+            for gate in circuit.queue:
+                circuit_stim.append(gate.__class__.__name__, list(gate.qubits))
+
+            quad_1, quad_2, quad_3, quad_4, x_phases, z_phases = (
```

Comment:
Maybe, instead of `quad_x`, you could use `x_stab`, `z_destab`, ..., to make it clearer what they are.

(or `xn`, `x2n`, ..., or whatever has a more direct connection).

Reply:
Done